superset-frontend/src/pages/DvtAlerts/index.tsx

```diff
@@ -0,0 +1,156 @@
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useAppSelector } from 'src/hooks/useAppSelector';
+import DvtPagination from 'src/components/DvtPagination';
+import DvtTable from 'src/components/DvtTable';
+import withToasts from 'src/components/MessageToasts/withToasts';
+import DvtButton from 'src/components/DvtButton';
+import DvtIconDataLabel from 'src/components/DvtIconDataLabel';
+import { StyledAlerts, StyledAlertsButton } from './dvt-alerts.module';
+
+function AlertList() {
+  const [apiData, setApiData] = useState([]);
+
+  const [page, setPage] = useState<number>(1);
+  const alertsSelector = useAppSelector(state => state.dvtSidebar.alerts);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('/api/v1/report/');
+        const data = await response.json();
+        console.log(data);
+        const editedData = data.result
+          .filter((item: any) => item.type === 'Alert')
+          .map((item: any) => ({
+            lastRun: new Date(item.last_eval_dttm).toLocaleString('tr-TR', {
+              day: '2-digit',
+              month: '2-digit',
+              year: 'numeric',
+              hour: '2-digit',
+              minute: '2-digit',
+              second: '2-digit',
+            }),
+            name: item.name,
+            schedule: new Date(item.created_on).toLocaleString('tr-TR', {
+              day: '2-digit',
+              month: '2-digit',
+              year: 'numeric',
+              hour: '2-digit',
+              minute: '2-digit',
+              second: '2-digit',
+            }),
+            notificationMethod: item.crontab_humanized,
+            createdBy: `${item.created_by.first_name} ${item.created_by.last_name}`,
+            owners: `${item.owners[0].first_name} ${item.owners[0].last_name}`,
+            modified: `${item.changed_by.first_name} ${item.changed_by.last_name}`,
+            status: item.last_state,
+            active: item.active.toString(),
+          }));
+
+        const filteredData = editedData.filter(
+          (item: any) =>
+            (alertsSelector.owner
+              ? `${item.owners[0].first_name} ${item.owners[0].last_name}` ===
+                alertsSelector.owner
+              : true) &&
+            (alertsSelector.status
+              ? item.last_state === alertsSelector.status
+              : true) &&
+            (alertsSelector.createdBy
+              ? `${item.created_by.first_name} ${item.created_by.last_name}` ===
+                alertsSelector.createdBy
+              : true),
+        );
+        setApiData(filteredData);
+      } catch (error) {
+        console.error('Hata:', error);
+      }
+    };
+
+    fetchData();
+  }, [alertsSelector]);
+  const modifiedData = {
```

Comment:
modifiedData react component function içinde olmamalı import altındaki taşınmalı

Reply:
ilk data hazır için bu header